includes/discovery/sensors/state/cisco.inc.php

```diff
@@ -24,6 +24,7 @@
     ['num_oid' => '.1.3.6.1.4.1.9.9.13.1.4.1.3.',       'oid' => 'ciscoEnvMonFanStatusTable',            'state_name' => 'ciscoEnvMonFanState',          'mib' => 'CISCO-ENVMON-MIB',                'descr' => 'ciscoEnvMonFanStatusDescr'],
     ['num_oid' => '.1.3.6.1.4.1.9.9.13.1.5.1.3.',       'oid' => 'ciscoEnvMonSupplyStatusTable',         'state_name' => 'ciscoEnvMonSupplyState',       'mib' => 'CISCO-ENVMON-MIB',                'descr' => 'ciscoEnvMonSupplyStatusDescr'],
     ['num_oid' => '.1.3.6.1.4.1.9.9.117.1.1.2.1.2.',    'oid' => 'cefcFRUPowerStatusTable',              'state_name' => 'cefcFRUPowerOperStatus',       'mib' => 'CISCO-ENTITY-FRU-CONTROL-MIB',    'descr' => 'Sensor Name'],
+    ['num_oid' => '.1.3.6.1.4.1.9.9.117.1.2.1.1.2.',    'oid' => 'cefcModuleOperStatus',                 'state_name' => 'cefcModuleOperStatus',         'mib' => 'CISCO-ENTITY-FRU-CONTROL-MIB',    'descr' => 'Sensor Name'],
```

Comment:
"Sensor name" is not a good description

Reply:
Hi!,

So `Sensor Name` is not used as the description. Its used as a placeholder until we get here: https://github.com/librenms/librenms/blob/cb06af670c854852b667a71c06ee0cc4f92c941e/includes/discovery/sensors/state/cisco.inc.php#L246

This part of the code looks up the index and gets the entPhysicalName from the entPhysical table and then appends 'Operating Status' to the string. 

FWIW, I copied the logic for this from the `cefcFRUPowerStatusTable` sensors as they are similar.